File: LorePdks.COMMON/DTO/Common/KisiTokenDTO.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace LorePdks.COMMON.DTO.Common
{
    public class KisiTokenDTO
    {
        public int id { get; set; }
        public int kisiId { get; set; }
        public string loginName { get; set; }
        public string token { get; set; }
        public string ipAdresi { get; set; }
        public string userAgent { get; set; }
        public DateTime? expDate { get; set; }
        public bool isLogin { get; set; }
        public KisiDTO kisiDto { get; set; }
    }
}

File: LorePdks.WEB/src/app/core/models/KisiTokenDto.ts
import { baseDto } from "./baseDto";
import { EkranDto } from "./EkranDto";
import { KisiDto } from "./KisiDto";
import { RolDto } from "./RolDto";

/**
 * Kullanıcı token bilgisini temsil eden model
 */
export class KisiTokenDto extends baseDto {
  loginName: string;
  token: string;
  ipAdresi: string;
  userAgent: string;
  expDate?: Date;
  isLogin: boolean;
  kisiDto?: KisiDto;
  ekranDtoList?: EkranDto[];
}

